source/docs/03-metadata.xml

```diff
@@ -1382,156 +1354,308 @@
-                  </p>
-                  <p>
-                     <figure>
-                        <head>Nesting of two bifolia and one folium</head>
-                        <graphic url="../images/ExampleImages/foliadesc-01.png"/>
-                     </figure>
-                  </p>
-                  <p>Multiple signatures (groups of nested pages) bound together can be reflected by encoding a sequence of <gi scheme="MEI">bifolium</gi> elements (with their respective contents). If the binding of a source is unknown, but <gi scheme="MEI">foliaDesc</gi> is needed for other reasons, it is recommended to use a sequence of <gi scheme="MEI">folium</gi> elements only, with no indication of nesting at all.</p>
-                  <div xml:id="linkingsurfaces" type="div4">
-                     <head>Linking surface elements</head>
-                     <p>The <gi scheme="MEI">surface</gi> element and it’s children are used to relate musical content with digitizations and specific image zones on them (see <ptr target="#facsimileElements"/>). <gi scheme="MEI">surface</gi> elements are always encoded in sequence within <gi scheme="MEI">facsimile</gi>, and thus lack the expressiveness of <gi scheme="MEI">foliaDesc</gi>. However, it is possible to relate these two concepts.</p>
-                     <p>
-                        <gi scheme="MEI">folium</gi> offers two specific attributes:</p>
+                  <div xml:id="textTitlePages" type="div4">
+                     <head>Title Pages</head>
+                     <p>A specialized element is furnished for the capture of titlepage information.</p>
                      <p>
                         <specList>
-                           <specDesc key="att.foliumSurfaces" atts="recto"/>
-                           <specDesc key="att.foliumSurfaces" atts="verso"/>
+                           <specDesc key="titlePage"/>
                         </specList>
                      </p>
-                     <p>These attributes are used to point to the <att>xml:id</att> of a <gi scheme="MEI">surface</gi> element.</p>
-                     <p>
-                        <figure>
-                           <head>Positional attributes on folium and bifolium</head>
-                           <graphic url="../images/ExampleImages/foliadesc-02.png"/>
-                        </figure>
-                     </p>
-                     <p>On <gi scheme="MEI">bifolium</gi>, the corresponding attributes are:</p>
+                     <p>The <gi scheme="MEI">titlePage</gi> element, modelled after a similar element in Encoded Archival Description (EAD), may occur within the textual matter preceding or following the musical content of the encoding. Since a diplomatic transcription of the titlepage is often necessary to accurately identify musical material contained within a source, <gi scheme="MEI">titlePage</gi> may also be used within the metadata header as a child of the <gi scheme="MEI">physDesc</gi> element.</p>
```

Comment:
```suggestion
                     <p>The <gi scheme="MEI">titlePage</gi> element, modelled after a similar element in the Encoded Archival Description (EAD), may occur within the textual matter preceding or following the musical content of the encoding. Since a diplomatic transcription of the title page is often necessary to accurately identify musical material contained within a source, the <gi scheme="MEI">titlePage</gi> element may also be used within the metadata header as a child of the <gi scheme="MEI">physDesc</gi> element.</p>
```